src/displays.jl

```diff
@@ -9,13 +9,19 @@ struct TKeyNode
 end
 function children(f::ROOTFile)
     # display TTrees recursively
+    # subsequent TTrees with duplicate fName will be skipped
+    # since TKey cycle number is guaranteed to be decreasing
     # then all TKeys in the file which are not for a TTree
+    seen = Set{AbstractString}()
```

Comment:
```suggestion
    seen = Set{String}()
```

we know `keys()` returns Strings I think